docs/howtoguides/trusty_legacy_support.rst

```diff
@@ -0,0 +1,105 @@
+its end of life support for esm-infra.
+
+
+Make sure ``ua`` is up-to-date
+===============================
+
+All systems come with ``ua`` pre-installed through the
+``ubuntu-advantage-tools`` package. To make sure that you're running the latest
+version of ``ua``, run the following commands:
+
+.. code-block:: bash
+
+    sudo apt update && sudo apt install ubuntu-advantage-tools
+
+
+Check the status of the services
+================================
+
+After you have attached to a Pro subscription and
+updated the ``ubuntu-advantage-tools`` package, you can check which services
+are enabled by running the following command:
+
+.. code-block:: bash
+
+    sudo ua status
+
+This will show you which services are enabled or disabled on your machine
+(output truncated for brevity):
+
+.. code-block:: text
+
+    SERVICE          ENTITLED  STATUS    DESCRIPTION
+    esm-infra        yes       enabled   Expanded Security Maintenance for Infrastructure
+    esm-infra-legacy yes       disabled  Expanded Security Maintenance for Infrastructure on Legacy Instances
+    livepatch        yes       enabled   Canonical Livepatch service
+
+If you are entitled to the service, you can enable it by running:
+
+
+.. code-block:: bash
+
+    sudo ua enable esm-infra-legacy
+
+
+What to do if esm-infra-legacy is not entitled
+===============================================
+
+If status is showing that you are not entitled to the service, you
+will need to buy access to the service. You can contact sales/support for help on that regard.
+
+Once you have bought support, you will need to run the following command
+on your machine:
+
+.. code-block:: bash
+
+    sudo ua refresh
+
+
+This command will refresh the contract definitions on your machine. After that, you
+can confirm that the service is now entitled by running:
+
+
+.. code-block:: bash
+
+    sudo ua status
+
+If you are now entitled to the service, you can enable it with:
+
+.. code-block:: bash
+
+    sudo ua enable esm-infra-legacy
+
+
+Trusty caveats
+===============
+
+There are some known caveats for the Trusty version of `ubuntu-advantage-tools`:
+
+* **Disabling esm-infra also disables esm-infra-legacy**: Even though this is happening, users
+  can disable esm-infra-legacy without consequences for esm-infra
+* **do-release-upgrade fails if esm-infra enabled and packages installed from service**: The
+  do-release-upgrade operation will fail here, as there will be an APT dependency chain issue
+  when performing the operation.
+
+Please note that all of these problems won't happen on Xenial forwards.
+The Ubuntu Pro Client is still fully supported on those later releases and
+these issues have already been fixed for them.
+
+
+Why 14.04 (Trusty) no longer receives new Ubuntu Pro Client features
+---------------------------------------------------------------------
+
+On 14.04 (Trusty), the Ubuntu Pro Client package is not receiving upstream
```

Comment:
Tell them why?
Before Legacy we thought there won't be a chance as well, we somehow have to admit that.

something like this maybe:
"For a further reduced risk of regressions on 14.04 (Trusty) the Pro client package is almost frozen. Hence it is not receiving regular upstream backports like newer Ubuntu LTS releases do. Beyond version 19.7 there won't be updates except any critical CVE maintenance or features required to deliver features explicitly targeted for Trusty like esm-infra-support in 2024".